src/Datadog.Trace/CorrelationIdentifier.cs

```diff
@@ -3,19 +3,53 @@
 namespace Datadog.Trace
 {
     /// <summary>
-    /// An API to access the active trace and span ids.
+    /// An API to access identifying values of the service and the active span
     /// </summary>
     public static class CorrelationIdentifier
     {
-        internal static readonly string ServiceNameKey = "dd.service";
-        internal static readonly string ServiceVersionKey = "dd.version";
+        internal static readonly string ServiceKey = "dd.service";
+        internal static readonly string VersionKey = "dd.version";
         internal static readonly string EnvKey = "dd.env";
         internal static readonly string TraceIdKey = "dd.trace_id";
         internal static readonly string SpanIdKey = "dd.span_id";
 
         /// <summary>
-        /// Gets the trace id
+        /// Gets the name of the service
         /// </summary>
+        public static string Service
+        {
+            get
+            {
+                return Tracer.Instance.DefaultServiceName ?? string.Empty;
+            }
+        }
+
+        /// <summary>
+        /// Gets the version of the service
+        /// </summary>
+        public static string Version
+        {
+            get
+            {
+                return Tracer.Instance.Settings.ServiceVersion ?? string.Empty;
+            }
+        }
+
+        /// <summary>
+        /// Gets the environment name of the service
+        /// </summary>
+        public static string Env
```

Comment:
Shouldn't we use the full name (Environment) here? I don't know what's the usual naming convention for the tracer

Reply:
`Env` is the canonical term we use to segment Datadog metrics by environment, so I think we should keep it as `Env`. Especially as the new API names will match the new `DD_SERVICE, `DD_ENV` and `DD_VERSION` environment variables that are being introduced across APM to correlate different metrics from one service